src/main/java/pl/zalwi/UsersController.java

```diff
@@ -0,0 +1,78 @@
+package pl.zalwi;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import pl.zalwi.data.User;
+import pl.zalwi.data.UserRepository;
+
+import java.util.List;
+
+@Controller
+public class UsersController {
+
+    UserRepository userRepository;
+
+    public UsersController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @RequestMapping("/")
+    public String addUserForm(){
+        return "redirect:/index.html";
+    }
+
+    @ResponseBody
+    @RequestMapping("/users")
+    public String listOfUsers(){
+        String result = "<b>Lista użytkowników</b><br>";
+        for(User user: userRepository.getAll()){
+            result += user + "<br>";
+        }
+        result += "Powrót do <a href='/'>strony głównej</a>";
+        return result;
+    }
+
+    @RequestMapping("/add")
+    public String addUser(@RequestParam String firstName, @RequestParam String lastName, @RequestParam int age){
+        if(firstName.length()<1){
```

Comment:
Jest metoda `isEMpty` którą można wywołać na Stringu, najlepiej wcześniej sprawadzając czy String nie jest nullem. Albo możesz wykorzysać bibliotekę `Apache Commons` w której masz klasę `StringUtils`, która robi to od razu.